daemon/daemon.go

```diff
@@ -267,15 +286,17 @@ func (d *Daemon) Shutdown() error {
 	}
 
 	logrus.Debugf("Start cleanup containerd...")
+	if err := d.ctrdClient.Cleanup(); err != nil {
+		errMsg = fmt.Sprintf("%s\n", err.Error())
+	}
 
-	if err := d.containerd.Cleanup(); err != nil {
+	if err := d.ctrdDaemon.Stop(); err != nil {
```

Comment:
If `d.ctrdClient.Cleanup()` returns non-nil as well, here we will overwrite the errMsg. I think this has some potential possibility to miss information. How about using string append way to still keep the message? @fuweid

Reply:
Yes. There is some issues about handing error. basically, I will update this part in following PR, not in this pr.